var/spack/repos/builtin/packages/py-cvxopt/package.py

```diff
@@ -25,6 +26,7 @@ class PyCvxopt(PythonPackage):
 
     # Required dependencies
     depends_on('python@2.7:')
+    depends_on('python@2.7:3.7.999', when='@:1.1.9')
```

Comment:
I think python should be `type=('build', 'link', 'run'))`

Reply:
Fixed in https://github.com/spack/spack/pull/19385/commits/3c9474263049eb207ea4b8455b92aa0592757f27